torch/_decomp/decompositions.py

```diff
@@ -1137,6 +1137,13 @@ def addmm(self: Tensor, mat1: Tensor, mat2: Tensor, beta: int = 1, alpha: int =
     return out + beta * self
 
 
+@register_decomposition(aten._int_mm)
+@out_wrapper()
+@pw_cast_for_opmath
+def _int_mm(self: Tensor, mat1: Tensor, mat2: Tensor):
```

Comment:
Will try removing this and post the resulting error.

Reply:
I don't understand what this decomp does, can you add a comment?

Reply:
It's really an identity decomposition. I will send a follow-up that removes it. Last time I tried removing it it caused exceptions within the stack.